dev/io.openliberty.org.apache.myfaces.4.0_fat/.classpath

```diff
@@ -10,6 +10,8 @@
 	<classpathentry kind="src" path="test-applications/MultipleInputFileTest.war/src"/>
 	<classpathentry kind="src" path="test-applications/AjaxRenderExecuteThisTest.war/src"/>
 	<classpathentry kind="src" path="test-applications/AcceptInputFileTest.war/src"/>
+	<classpathentry kind="src" path="test-applications/ProgrammaticFaceletTests.war/src"/>
+	<classpathentry kind="src" path="test-applications/WebSocket.war/src"/>
```

Comment:
Is the WebSocket.war/src actually part of this PR? I didn't see it.

Reply:
Correct. This will be part of the websocket PR.